products/llbuild-analyze/Sources/CriticalPathTool.swift

```diff
@@ -6,95 +6,58 @@
 // See http://swift.org/LICENSE.txt for license information
 // See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 
-import TSCUtility
 import TSCBasic
 import llbuildAnalysis
 import llbuildSwift
+import ArgumentParser
 
 import struct Foundation.Data
-import class Foundation.ListFormatter
 import class Foundation.FileManager
 import class Foundation.JSONEncoder
 
-public final class CriticalPathTool: Tool<CriticalPathTool.Options> {
+struct CriticalPathTool: ParsableCommand {
+    static var configuration = CommandConfiguration(commandName: "critical-path", shouldDisplay: true)
```

Comment:
You can omit the `shouldDisplay` parameter here.
```suggestion
    static var configuration = CommandConfiguration(commandName: "critical-path")
```